nebullvm/pipelines/steps.py

```diff
@@ -659,7 +657,7 @@ def build_pipeline_from_model(
         logger (Logger, optional): Logger defined by the user.
     """
     compressor_step = _get_compressor_step(
```

Comment:
why don't we pass the metric information anymore?

Reply:
Because it was used only on the check that I removed that prevented the compressor to be used when the metric was none. Now if metric_drop_ths is set the metric is always != None, so it's not necessary anymore